compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/KernelDispatch.h

```diff
@@ -23,7 +23,7 @@ enum class AIEPassPipeline : int32_t {
 };
 
 /// Enum for target tiling op.
-enum class TilingOp : int32_t { Matmul = 0, RhsCopy = 1, LhsCopy = 2 };
+enum class PeelLoopType : int32_t { First = 0, Last = 1, FirstLast = 2 };
```

Comment:
I see there were no uses of `TilingOp` enum anyway.

With regards to `PeelLoopType` change the preceding comment.